e2e/init.js

```diff
@@ -0,0 +1,13 @@
+import detox from "detox";
+import { detox as config } from "../package.json";
```

Comment:
What's the benefit of having detox config in `package.json`?

Reply:
I could move it to its own config file, I'd put it in package.json because the jest config was already there.